docs/en/operations/userspace-page-cache.md

```diff
@@ -0,0 +1,76 @@
+---
+description: 'caching mechanism that allows for caching of 
+data in in-process memory rather than relying on the OS page cache.'
+sidebar_label: 'Userspace page cache'
+sidebar_position: 65
+slug: /operations/userspace-page-cache
+title: 'Userspace page cache'
+---
+
+# Userspace page cache
+
+## Overview {#overview}
+
+> The userspace page cache is a new caching mechanism that allows for caching of 
+data in in-process memory rather than relying on the OS page cache.
+
+ClickHouse already offers the [Filesystem cache](/docs/operations/storing-data) 
+as a way of caching on top of remote object storage such as Amazon S3, Google 
+Cloud Storage (GCS) or Azure Blob Storage. The userspace page cache is designed 
+to speed up access to remote data when the normal OS caching isn't doing a good 
+enough job. 
+
+It differs from the filesystem cache in the following ways:
+
+| Filesystem Cache                                        | Userspace page cache                  |
+|---------------------------------------------------------|---------------------------------------|
+| Writes data to the local filesystem                     | Present only in memory                |
+| Takes up disk space (configurable on tmpfs)             | Independent of filesystem             |
```

Comment:
About tmpfs - emphasize that this is an unusual option:

"however, can be configured on tmpfs"
or
"also configurable on tmpfs"